pandasai/connectors/databricks.py

```diff
@@ -0,0 +1,69 @@
+"""
+Databricks Connector to connects you to your Databricks SQL Warhouse on
+Azure, AWS and GCP
+"""
+
+import os
+from .base import BaseConnectorConfig, DatabricksConnectorConfig
+from sqlalchemy import create_engine
+from typing import Union
+from .sql import SQLConnector
+
+
+class DatabricksConnector(SQLConnector):
+    """
+    Databricks connectors are used to connect to Databricks Data Cloud.
+    """
+
+    def __init__(self, config: DatabricksConnectorConfig):
```

Comment:
This accepts also a dictionary, `Union[DatabricksConnectorConfig, dict]`. In that case, we need to convert it to a DatabricksConnectorConfig.